src/Compilers/Core/Portable/Symbols/Nullability.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis
+{
+    public enum Nullability
+    {
+        Unknown = 0,
+        NonNull = 1,
+        MaybeNull = 2
+    }
```

Comment:
We should document this `enum`, perhaps including examples. For instance, what is the nullability of a reference to an unconstrained type parameter `T`? What is the nullability of `int?`?